docs/reference/cfg/annotate/index.html

```diff
@@ -475,6 +475,29 @@
 
 
 
+<ul class="td-sidebar-nav__section pr-md-3">
+  <li class="td-sidebar-nav__section-title">
+    <a  href="/kpt/reference/cfg/delete-setter/" class="align-left pl-0 pr-2 collapsed td-sidebar-link td-sidebar-link__section">delete-setter</a>
+  </li>
+  <ul>
+    <li class="collapse " id="kptreferencecfgdelete-setter">
+      
```

Comment:
Can you remove these and the following empty lines?

Reply:
This is code generated by Hugo. We don't want to start making changes here.